### PR TITLE
Lesson 1 Homework

### PR DESCRIPTION
**Pull request by zanilee**

### CODE REVIEW

**Comment by puzankov** [comment]
Move all code to lesson01 folder

File: app.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var express = require('express');
+
+var app = express();
+
+var path = require('path');
+
+app.use('/', express.static(path.join(__dirname, '/client_src')));
+
+app.listen(3000, function () {
+  console.log('Server started');
```
Comment (puzankov) [inline app.js:10]: Server started on http://localhost:3000/

File: client_src/posts.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        for(var i = 0, j = context.length; i < j; i++) {
+            var stripeClass = '';
+
+            if((i % 2) == 0) {
+                stripeClass = '_even';
+            } else {
+                stripeClass = '_odd';
+            }
+
```
Comment (puzankov) [inline client_src/posts.js:37]: var stripeClass = (i % 2) ? '_even' : '_odd';

File: client_src/posts.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    //
+    //  Block helper
+    //  ---------------------------------------------
+
+    var postTableTemplateRaw = $('#posts-table-template').html();
+    var postsTableTemplate = Handlebars.compile(postTableTemplateRaw);
+
+    Handlebars.registerHelper('table', function(context, options) {
+        var result = "<ul>";
```
Comment (puzankov) [inline client_src/posts.js:27]: '<ul>'

**Comment by zanilee** [comment]
Everything fixed in b2cc30b5

**Comment by puzankov** [comment]
+1

**Pull request merged**